junit-platform-console/src/main/java/org/junit/platform/console/options/AvailableOptions.java

```diff
@@ -70,8 +73,18 @@
 		disableAnsiColors = parser.accepts("disable-ansi-colors",
 			"Disable ANSI colors in output (not supported by all terminals).");
 
-		hideDetails = parser.accepts("hide-details",
-			"Hide details while tests are being executed. Only show the summary and test failures.");
+		// TODO Remove "hide-details" before final release. Only here to make "legacy" Gradle plugin happy.
+		// If removed, an old Gradle:junitPlatformTest will report:
+		//   Exception in thread "main" joptsimple.UnrecognizedOptionException: hide-details is not a recognized option
+		parser.accepts("hide-details", "@Deprecated. Use '--details none' instead.");
+
+		List<String> names = stream(Details.values()).map(d -> d.name().toLowerCase()).collect(toList());
```

Comment:
Of course. ;)

Although that raises a good question: do we want to call `Details.toString` or `Details.name` on each details instance?

I don't suppose it really matters for now, and if we do need to use `Details.name()`, we can always switch back to streams.